yarn2-plugins/prune-dev-dependencies/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "yarn-plugin-prune-dev-dependencies",
```

Comment:
If this works, we'll need to publish this package so that we can use it in `heroku/buildpacks-nodejs`. Maybe add a `@heroku` prefix to this name, since that's where we're hosting other packages?